### DISCUSSION TITLE
Where can i check the support/EOL dates for Kong OSS versions ?

### DISCUSSION THREAD

**Discussion by shettyh** [Kong/kong#10866] [Help] [answered]
[Here](https://docs.konghq.com/gateway/latest/support-policy/) i can see support dates for enterprise versions only, is this also applicable for OSS releases ?

**Comment by hanshuebner** [comment, marked as answer]
Hello @shettyh,

Kong OSS, we only provide support for the latest release.  We're not generally backporting changes to older versions, but sometimes re-release on a best-effort basis.  Long-term support policies are only available for Kong Gateway Enterprise.

Kind regards, Hans

**Reply by shettyh** [reply]
@hanshuebner thanks for response.
